spec/swift/app_spec.rb

```diff
@@ -442,6 +468,13 @@ def app
           last_response.headers["ETag"].must_equal "\"f9f85fbf5aa1fa378fd79ac8aa0a457d\""
         end
 
+        it "has a Cache-Control in the header" do
```

Comment:
I think the language can be improved a bit here. E.g. "returns a Cache-Control header" or "includes Cache-Control in the headers" or similar.